### DISCUSSION TITLE
wanted to know the command to run the grule file

### DISCUSSION THREAD

**Discussion by nagarajtrivedi** [hyperjumptech/grule-rule-engine#209] [General]
Hi, I wanted to know what is the command to execute a grule file.
It is a bit urgent. Please reply soon.

Thanks
Nagaraj

**Comment by jinagamvasubabu** [comment]
Hello @nagarajtrivedi, You cannot execute a grule file directly. you can load the rules to the knowledgebase and pass a fact to check against the rules and see if its matching 

Refer to any existing tests under examples to see how to use it. https://github.com/hyperjumptech/grule-rule-engine/blob/master/examples/AgeCheckSample_test.go

**Reply by nagarajtrivedi** [reply]
OK I will check it
